### PR TITLE
Add a default throttle-size of 1 to the throttle component.

### PR DESCRIPTION
**Pull request by glenna-m**

### CODE REVIEW

**Comment by dbooth-boston** [comment]
LGTM

File: components/throttle.js [hunk 1]
```diff
@@ -39,15 +39,13 @@ module.exports = wrapper(throttler);
  * @return a hash with the next elements to be processed.
  */
 function throttler(throttle_size, file_envvar, encoding) {
-
     var node = this.nodeInstance;
     if (_.isUndefined(node.throttle)) node['throttle'] = {}; 
 
     // Get the number of hash elements being requested
     var throttleSize = parseInt(throttle_size);
```
Comment (catch-point) [inline components/throttle.js]: This could also be written as var throttleSize = _.isFinite(throttle_size) ? +throttle_size : 1;

**Comment by catch-point** [comment]
LGTM

File: components/throttle.js (hunk 1 above)
Comment (glenna-m) [inline components/throttle.js]: That does look a little neater.  I can make that change and commit.

**Pull request merged**